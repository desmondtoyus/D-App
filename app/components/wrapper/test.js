import React from 'react'
import {renderWithProviders, cleanup} from '../../../__tests__/setup/test-setup'
import Wrapper from './index'

 afterEach(cleanup)

 describe('<MainWrapper />', () => {
    it('Should match a snapshot', () => {
        const { asFragment } = renderWithProviders(<Wrapper desc={'Testing'}/>)
        expect(asFragment).toMatchSnapshot()
    });
    it('Should contain Header', () => {
        const { getByTestId } = renderWithProviders(<Wrapper desc={'Testing'} />);
        const appHeader = getByTestId('header');
        expect(appHeader).toBeTruthy();
      });
 })
