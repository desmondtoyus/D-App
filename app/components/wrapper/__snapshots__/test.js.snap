// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainWrapper /> Should match a snapshot 1`] = `[Function]`;

exports[`<MainWrapper /> should take a snapshot 1`] = `[Function]`;
